src/ztc/datamodel/models/eigenschap.py

```diff
@@ -208,7 +208,10 @@ def clean(self):
         """
         super().clean()
 
-        if bool(self.specificatie_van_eigenschap) ^ bool(self.referentie_naar_eigenschap):  # xor
+        if self.specificatie_van_eigenschap and self.referentie_naar_eigenschap:
```

Comment:
I kind of liked the XOR happening here instead of having to do it manually. You can still use the original line of code with a slight modification.